usability-analysis.md

```diff
@@ -1,51 +1,47 @@
-# Usability Analysis
+# Usability analysis
 
-In order to use or even further develop existing open-source software, it must first undergo a usability analysis.
-All free software should undergo this usability analysis, regardless of whether it is to be used as a framework, library, [development tools](./in-house-development.html) or [use](./use.html) as ready-made software.
+In order to use or even further develop existing free software, it must first be subjected to a usability analysis.
```

Comment:
I think the translation here is wrong, `open-source` is the better term here.

Reply:
Its free software as in free speech, not as in free beer.